src/cmd/singularity/cli/sif.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/singularityware/singularity/src/pkg/sylog"
+
+	"github.com/singularityware/singularity/src/pkg/buildcfg"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	sifCmd.Flags().SetInterspersed(false)
+	singularityCmd.AddCommand(sifCmd)
+
+}
+
+var sif = buildcfg.SBINDIR + "/sif"
```

Comment:
I don't think this needs to be a global. It could be inside the `Run` `func()` below.